conans/model/conf.py

```diff
@@ -69,6 +69,7 @@
     "tools.deployer:symlinks": "Set to False to disable deployers copying symlinks",
     "tools.files.download:retry": "Number of retries in case of failure when downloading",
     "tools.files.download:retry_wait": "Seconds to wait between download attempts",
+    "tools.graph:skip_binaries": "Allow binary skipping in the graph",
```

Comment:
I think it would be nice to complete the description mentioning what the default value for this configuration is.